src/PlusDataCollection/BkProFocus/vtkPlusBkProFocusOemVideoSource.cxx

```diff
@@ -1256,18 +1256,23 @@ PlusStatus vtkPlusBkProFocusOemVideoSource::AddParametersToFrameFields()
 {
   vtkPlusUsDevice::InternalUpdate();// Move to beginning of vtkPlusBkProFocusOemVideoSource::InternalUpdate()?
 
-  this->FrameFields[KEY_PROBE_TYPE]   = PlusCommon::ToString(this->GetProbeType());
-  this->FrameFields[KEY_ORIGIN]       = PlusCommon::ToString(this->CalculateOrigin());
-  this->FrameFields[KEY_ANGLES]       = PlusCommon::ToString(this->CalculateAngles());
-  this->FrameFields[KEY_BOUNDING_BOX] = PlusCommon::ToString(this->CalculateBoundingBox());
-  this->FrameFields[KEY_DEPTHS]       = PlusCommon::ToString(this->CalculateDepths());
-  this->FrameFields[KEY_LINEAR_WIDTH] = PlusCommon::ToString(this->CalculateLinearWidth());
-
-  this->FrameFields[KEY_SPACING_X]    = PlusCommon::ToString(this->GetSpacingX());
-  this->FrameFields[KEY_SPACING_Y]    = PlusCommon::ToString(this->GetSpacingY());
-
-  this->FrameFields[KEY_DEPTH]        = PlusCommon::ToString(this->CalculateDepthMm());
-  this->FrameFields[KEY_GAIN]         = PlusCommon::ToString(this->CalculateGain());
+  this->FrameFields[KEY_PROBE_TYPE]   = PlusCommon::ToString<int>(this->GetProbeType());
+  std::string output;
+  PlusCommon::JoinTokensIntoString(this->CalculateOrigin(), output, ',');

```

Comment:
I'm thinking that your default separator (space) is much better than my comma. This is because comma may sometimes be used in float numbers. So just change the separator back to the default or set space explicitly here.